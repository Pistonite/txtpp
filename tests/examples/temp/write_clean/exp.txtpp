This example writes to a temp file, then include it
 +TXTPP#temp temp.out
 + this is content
 + in the temp.out file
This line should be in the output
Now we include it
TXTPP#include temp.out
