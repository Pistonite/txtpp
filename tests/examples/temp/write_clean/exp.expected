This example writes to a temp file, then include it
This line should be in the output
Now we include it
 this is content
 in the temp.out file
